prow/config.yaml

```diff
@@ -6107,6 +6107,22 @@ periodics:
       - "--repo=k8s.io/autoscaler=master"
       - "--root=/go/src"
       - "--timeout=60"
+      env:
+      - name: DOCKER_IN_DOCKER_ENABLED
+        value: true
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - name: docker-graph
+        mountPath: /docker-graph
+      # port set to limit to one job per node since sharing docker isn't safe
+      ports:
+      - containerPort: 9999
+        hostPort: 9999
+    volumes:
+    - name: docker-graph
+      emptyDir: {}
```

Comment:
actually, @BenTheElder this should be `var-lib-docker` or `docker-graph`?

Reply:
doesn't matter, just need to be mounted to /docker-graph for our image